samples/apps/copilot-chat-app/webapi/DeploymentTemplates/sk-existing-ai.json

```diff
@@ -0,0 +1,331 @@
+{
```

Comment:
Do we need to check this in? I thought it was generated by bicep.

Reply:
It is generated bi bicep but, unfortunately, the "Deploy to Azure" buttons only support ARM templates